libs/sdk-ui-charts/src/highcharts/chart/HighChartsRenderer.tsx

```diff
@@ -97,6 +97,8 @@ export default class HighChartsRenderer extends React.PureComponent<
         this.setChartRef = this.setChartRef.bind(this);
         this.onLegendItemClick = this.onLegendItemClick.bind(this);
         this.throttledOnWindowResize = throttle(this.onWindowResize.bind(this), 100);
+        this.onZoomOutButtonClick = this.onZoomOutButtonClick.bind(this);
+        this.onChartSelection = this.onChartSelection.bind(this);
```

Comment:
no need to bind if you use arrow function

Reply:
Fixed!